openwisp_radius/api/views.py

```diff
@@ -487,18 +492,19 @@ def get_serializer_context(self):
             return
         uid = user_pk_to_url_str(user)
         token = default_token_generator.make_token(user)
-        password_reset_urls = app_settings.PASSWORD_RESET_URLS
-        default_url = password_reset_urls.get('default')
         domain = get_current_site(self.request).domain
         if getattr(self, 'swagger_fake_view', False):
-            organization_pk, organization_slug = None, None  # pragma: no cover
+            organization_slug = None  # pragma: no cover
         else:
-            organization_pk = self.organization.pk
             organization_slug = self.organization.slug
-        password_reset_url = password_reset_urls.get(str(organization_pk), default_url)
-        password_reset_url = password_reset_url.format(
-            organization=organization_slug, uid=uid, token=token, site=domain
-        )
+        password_reset_url = get_password_reset_url(self.organization)
```

Comment:
I would make `get_password_reset_url` a method of this class or a function in this file. I think it makes sense as a method of this class.